tests/fixture/mod.rs

```diff
@@ -0,0 +1,128 @@
+use git2;
+use git_rsl::{self, BranchName, RemoteName};
+use tempfile::{self, TempDir};
+use assert_cli::{Assert, Environment};
+
+#[allow(dead_code)]
+pub struct TestFixture {
+    origin: TempDir,
+    local: TempDir,
+    gpg_home: TempDir,
+    pub env: Environment
+}
+
+use std::fs::File;
+use std::io::Write;
+
+fn write_gpg_keygen_script(out_path: &str) {
+    let mut f = File::create(out_path).unwrap();
+    f.write_all("%no-protection\n\
+                 %transient-key\n\
+                 Key-Type: DSA\n\
+                 Key-Length: 1024\n\
+                 Name-Email: test_user@polysync.io\n\
+                 Expire-Date: 0\n\
+                 %commit\n".as_bytes()
+    ).unwrap();
+}
+
+use std::process::Command;
+fn first_gpg_key_id(gpghome: &str) -> String {
+    let out = Command::new("gpg2")
+        .env("GNUPGHOME", gpghome)
+        .args(vec!["--with-colons", "--list-keys"])
+        .output()
+        .unwrap();
+
+    // This splits out lines of this form:
+    //    "pub:u:1024:17:31BC51B4BB930CA9:1528761082:::u:::scaSCA:::::::"
+    // Keep the one that starts with 'pub', and take the 5th field.
+
+    String::from_utf8_lossy(&out.stdout)
+        .lines().filter(|l| l.starts_with("pub"))
+        .next().unwrap()
+        .split(":")
+        .skip(4)
+        .next().unwrap().to_string()
+}
+
+
+pub fn run(e: &Environment, args: &[&str]) {
+    Assert::command(args).with_env(e).unwrap()
+}
+
+pub fn runv(e: &Environment, lines: &[&[&str]]) {
+    for line in lines {
+        run(e, line);
+    }
+}
+
+use std::env;
```

Comment:
Similarly, either move `use` into the scope where it is employed or at the file/mod top.